src/colorizer/ColorizeCanvas.ts

```diff
@@ -167,7 +167,7 @@ export default class ColorizeCanvas {
   }
 
   setFeature(name: string): void {
-    if (!this.dataset?.features.hasOwnProperty(name)) {
+    if (!this.dataset || !Object.prototype.hasOwnProperty.call(this.dataset.features, name)) {
```

Comment:
ouch, this got worse.  Is there anything more concise than `Object.prototype.hasOwnProperty.call`?  Should `dataset.features` be typed such that it has a name property?

Reply:
This was flagged by eslint! From what I understand, it's because it's possible to make non-default-prototype Objects which may have undefined behavior for `hasOwnProperty`. (https://eslint.org/docs/latest/rules/no-prototype-builtins).

I could just undo this change and override the error since we know `dataset` is either null or the `Dataset` type, would that make sense?

Reply:
I don't think it's worth it.  I'm ok with it the way it is. The real problem is not with the Dataset type, it's that you truly have to call hasOwnProperty.  I suppose you could do something like "Object.keys includes name" but I don't know if you'd run into the same problem.

Reply:
Looks like Object.keys().includes() works here!